modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js

```diff
@@ -127,11 +130,28 @@ export default withRouter(
 
 		useEffect(() => {
 			getThread(questionId, context.siteKey)
-				.then(({data: {messageBoardThreadByFriendlyUrlPath}}) => {
-					setQuestion(messageBoardThreadByFriendlyUrlPath);
+				.then(
```

Comment:
In the future we should improve the existing error boundary... to do something like this automatically

Reply:
Yes. I think it can be understood both ways. It is not the same an uncontrolled error that a query that returns an error, so in some cases it is good to differentiate, but it is confusing to check the error once the promised is resolved correctly.